C/c4Database.cc

```diff
@@ -377,6 +377,22 @@ bool c4db_purgeDoc(C4Database *database, C4Slice docID, C4Error *outError) {
     return false;
 }
 
+uint64_t c4db_nextDocExpiration(C4Database *database)
+{
+	KeyStore expiryKvs(database, "expiry");
+	DocEnumerator e(expiryKvs);
+	while (e.next()) {
```

Comment:
Ah, right, I forgot that even if it is semantically a number it is stored as a "string" type.  Cool!